src/Cake.Common.Tests/Unit/IO/FileAliasesTests.cs

```diff
@@ -200,6 +202,22 @@ public void Should_Copy_File()
                 fixture.TargetFiles[0].Received(1).Copy(
                     Arg.Is<FilePath>(p => p.FullPath == "/Working/target/file1.txt"), true);
             }
+
+            [Fact]
+            public void Should_Log_Verbose_Message_With_Correct_Target()
+            {
+                // Given
+                var fixture = new FileCopyFixture();
+
+                // When
+                FileAliases.CopyFile(fixture.Context, "./file1.txt", "./target/file1.txt");
```

Comment:
Sorry for nitpicking, but a better test would be to test for `./file1.txt` and `./target/file2.txt`.
If I understand correctly the bug will output the filename of the target file first, which would give the same results.